src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -4,12 +4,17 @@ define(function (require) {
   var moment = require('moment');
 
   // Used only by the savedDashboards service, usually no reason to change this
-  module.factory('SavedDashboard', function (courier) {
+  module.factory('SavedDashboard', function (courier, Private) {
+    var BoundToConfigObj = Private(require('ui/bound_to_config_obj'));
 
     // SavedDashboard constructor. Usually you'd interact with an instance of this.
     // ID is option, without it one will be generated on save.
     _.class(SavedDashboard).inherits(courier.SavedObject);
     function SavedDashboard(id) {
+      var defaultDarkThemeConfig = new BoundToConfigObj({
```

Comment:
The reason I suggested using `BoundToConfigObj` is because it will keep the bound properties up to date when they change. This code dereferences the pattern on line 37 though, so subsequent updates to the config will not update the dashboards that have already been created. If instead the entire defaults object was a `BoundToConfigObj` then the existing values would get the updated defaults.